### PR TITLE
Add guide on monitor custom schedules

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do?

Add a guide for the new monitor custom schedules feature.

### Motivation

[DOCS-5035](https://datadoghq.atlassian.net/browse/DOCS-5035)

### Additional Notes

Do not merge. Pending final approval from PM

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-5035]: https://datadoghq.atlassian.net/browse/DOCS-5035?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/monitors/configuration/_index.md [hunk 1]
```diff
@@ -78,20 +78,24 @@ A cumulative time window has a fixed starting point and expands over time. Monit
...
 
 | Evaluation Window Ranges        | Evaluation Frequency  |
 |---------------------------------|-----------------------|
 | window < 24 hours               | 1 minute              |
 | 24 hours <= window < 48 hours   | 10 minutes            |
 | window >= 48 hours              | 30 minutes            |
 
+The evaluation frequency can also be configured so that the alerting condition of the monitor is checked on a daily, weekly, or monthly basis. In this configuration, the evaluation frequency is no longer dependent on the evaluation window, but on the configured schedule. 
+
+For more information see the guide on how to [Customize monitor evaluation frequencies][4].
```
Comment (urseberry) [inline content/en/monitors/configuration/_index.md:97]:
```suggestion
For more information, see the guide on how to [Customize monitor evaluation frequencies][4].
```

File: content/en/monitors/guide/custom_schedules.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+- link: "/monitors/downtimes"
+  tag: "Documentation"
+  text: "Downtimes"
+- link: "/monitors/configuration/?tab=thresholdalert#evaluation-window"
+  tag: "Documentation"
+  text: "Cumulative time windows"
+---
+
+{{< beta-callout url="#" btn_hidden="true" >}}
+Monitor custom schedule is in private beta. To request access, contact Support at support@datadoghq.com.
```
Comment (urseberry) [inline content/en/monitors/guide/custom_schedules.md:18]: Is email the best way? I've most often seen links to the help page (https://docs.datadoghq.com/help/) when we ask people to contact Support.

Comment (urseberry) [inline content/en/monitors/guide/custom_schedules.md:18]:
```suggestion
The monitor custom schedule feature is in private beta. To request access, contact Support at support@datadoghq.com.
```

File: content/en/monitors/guide/custom_schedules.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+{{< /beta-callout >}}
+
+## Overview 
+
+Set a specific evaluation time and control the evaluation frequency for monitors to track the execution of critical jobs running in your environment. Monitor Custom Schedules enables you to alert on systems and processes that do not need to be continuously monitored, such as cron jobs.
```
Comment (urseberry) [inline content/en/monitors/guide/custom_schedules.md:23]:
```suggestion
Set a specific evaluation time and control the evaluation frequency for monitors to track the execution of critical jobs running in your environment. Monitor Custom Schedules enable you to alert on systems and processes that do not need to be continuously monitored, such as cron jobs.
```

File: content/en/monitors/guide/custom_schedules.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+Monitor Custom Schedules is supported on events, logs, and metrics monitors with daily, weekly, and monthly scheduling intervals.
```
Comment (urseberry) [inline content/en/monitors/guide/custom_schedules.md:25]:
```suggestion
Monitor Custom Schedules are supported on events, logs, and metrics monitors with daily, weekly, and monthly scheduling intervals.
```

File: content/en/monitors/guide/custom_schedules.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+## Configuration
+
+{{< img src="/monitors/guide/custom_schedules/add_custom_schedule.png" alt="Button to add custom schedule in the monitor configuration" style="width:100%;" >}}
+
+Click **+ Add Custom Schedule** to configure your evaluation frequency.
```
Comment (urseberry) [inline content/en/monitors/guide/custom_schedules.md:31]:
```suggestion
Click **Add Custom Schedule** to configure your evaluation frequency. 
```

File: content/en/monitors/guide/custom_schedules.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+Click **+ Add Custom Schedule** to configure your evaluation frequency. 
+
+<div class="alert alert-warning">After a custom schedule has been enabled on a monitor, it cannot be disabled.
+</div>
+
+{{< tabs >}}
+{{% tab "Day" %}}
+Select the time of the day you want the monitor to evaluate at.
+
+For example, the following monitor checks every day at 8:00PM that a success event has been generated for each database instance as a result of the daily backup job.
```
Comment (urseberry) [inline content/en/monitors/guide/custom_schedules.md:40]:
```suggestion
For example, the following monitor checks every day at 8:00PM that the daily backup job generated a success event for each database instance. 
```

File: content/en/monitors/guide/custom_schedules.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+For example, the following monitor checks every week on Tuesday and Saturday at 6:00AM that marketing emails have been sent for each individual campaign. 
+
+{{< img src="monitors/guide/custom_schedules/custom_week.png" alt="Monitor configuration to check every week on Tuesday and Saturday at 6am that marketing emails have been sent for each individual campaign" style="width:100%;" >}}
+
+{{% /tab %}}
+
+{{% tab "Month" %}}
+Select the day of the month as well as the time of the day you want the monitor to evaluate at.
+
+For example, the following monitor checks on the first day of each month whether the cron-job generating customer invoices has successfully run.
```
Comment (urseberry) [inline content/en/monitors/guide/custom_schedules.md:58]:
```suggestion
For example, the following monitor checks on the first day of each month whether the cron job generating customer invoices has successfully run.
```

File: content/en/monitors/guide/custom_schedules.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+{{< img src="monitors/guide/custom_schedules/custom_month.png" alt="Monitor configuration to check on the first day of each month whether the cron-job generating customer invoices has successfully run." style="width:100%;" >}}
```
Comment (urseberry) [inline content/en/monitors/guide/custom_schedules.md:60]:
```suggestion
{{< img src="monitors/guide/custom_schedules/custom_month.png" alt="Monitor configuration to check on the first day of each month whether the cron- ob generating customer invoices has successfully run." style="width:100%;" >}}
```

File: content/en/monitors/guide/custom_schedules.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## RRULES
+
+Recurrence rule (RRULE) is a property name from [iCalendar RFC][1], which is the standard for defining recurring events. Use the [official RRULE generator][2] as a tool to generate recurring rules. Leverage RRULEs to cover more advanced scheduling use-cases.
```
Comment (urseberry) [inline content/en/monitors/guide/custom_schedules.md:67]:
```suggestion
Recurrence rule (RRULE) is a property name from the [iCalendar RFC][1], which is the standard for defining recurring events. Use the [official RRULE generator][2] to generate recurring rules. Leverage RRULEs to cover more advanced scheduling use cases. 
```

File: content/en/monitors/guide/custom_schedules.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+
+#### Example: Monitor evaluates every other month on the first and last Sunday of the month:
+
+```text
+FREQ=MONTHLY;INTERVAL=2;BYDAY=1SU,-1SU
+```
+
+{{< img src="monitors/guide/custom_schedules/RRULE_month_last_sunday.png" alt="RRULE syntax used in the UI to evaluate every other month on the first and last Sunday of the month" style="width:90%;" >}}
+
+## Alerting behavior of monitors with custom schedule
```
Comment (urseberry) [inline content/en/monitors/guide/custom_schedules.md:89]:
```suggestion
## Alerting behavior of monitors with custom schedules
```

File: content/en/monitors/guide/custom_schedules.md [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+
+Monitors using default scheduling run the query with the default evaluation frequency and send alerts based on monitor status transitions (for example, when a monitor goes from WARN to OK or from OK to ALERT).
+
```
Comment (urseberry) [inline content/en/monitors/guide/custom_schedules.md:92]: The timeline below illustrates the behavior of a monitor with default scheduling.  Note that the monitor sends alerts corresponding to status changes.

File: content/en/monitors/guide/custom_schedules.md [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+{{< img src="monitors/guide/custom_schedules/alerting_behavior_regular.png" alt="Visual diagram showing a when a monitor sends an alert based on monitor state transitions for the default schedule with a thirty minute evaluation frequency" style="width:100%;" >}}
+
+Monitors with custom schedules, on the other hand, evaluate on a daily, weekly, or monthly basis and send alerts based on the results of individual evaluations. Each evaluation is independent from the previous one and sends a notification when the result is not OK.
+
```
Comment (urseberry) [inline content/en/monitors/guide/custom_schedules.md:96]:
```suggestion
The timeline below illustrates the behavior of a monitor running on a custom schedule. Unlike the default scheduled monitor, the custom scheduled monitor sends an alert during its evaluation time based on the monitor state.
```

**Comment by estherk15** [comment]
Thanks for the review @urseberry, applied all your suggestions!

**Review by urseberry — APPROVED** [review]

**Pull request merged**